proto/stream.proto

```diff
@@ -1,17 +1,24 @@
-syntax = "proto2";
+syntax = "proto3";
 
 package pb;
 
-import "metadata.proto";
-import "source.proto";
-
+import "fee.proto";
+import "file.proto";
 
 message Stream {
-    enum Version {
-        UNKNOWN_VERSION = 0;
-        _0_0_1 = 1;
-    }
-    required Version version = 1;
-    required Metadata metadata = 2;
-    required Source source = 3;
+    bytes hash = 1;
+    string language = 2;
+    string title = 3;
+    string author = 4;
```

Comment:
this field should die entirely

Reply:
should it be changed to creator? if im publishing public-domain work that i did not create, how do i note that?

Reply:
Fair, something like this probably needs to exist. But if we're supporting that kind of thing we'd still want that to be a first-class entity (at some point, we're obviously not too close to that).

I'd recommend keeping this out of the 15 because I do not think a single plain text field will be the ultimate solution for how distinguishing authorship.